.github/workflows/milestone.yml

```diff
@@ -15,6 +15,10 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - uses: ./.github/actions/setup
+        timeout-minutes: 10
+        with:
+          java-version: 22
```

Comment:
before the change, what was it using?

Reply:
I believe that one that came by default with a worker. Let me check

Reply:
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#java

JDK 17